src/mux_client_impl/fd.rs

```diff
@@ -0,0 +1,30 @@
+use std::fs::File;
+use std::fs::OpenOptions;
+
+use std::os::unix::io::{AsRawFd, FromRawFd, IntoRawFd, RawFd};
+
+use once_cell::sync::OnceCell;
+
+/// Open "/dev/null" with RW.
+fn get_null_fd() -> RawFd {
+    static NULL_FD: OnceCell<RawFd> = OnceCell::new();
+    *NULL_FD.get_or_init(|| {
+        let file = OpenOptions::new()
+            .read(true)
+            .write(true)
+            .open("/dev/null")
+            .unwrap();
```

Comment:
This should, at the very least, be an `expect`. But I think better yet this function should be made to return a `Result`.

Reply:
I have updated the code to use `Result` and `expect` for "copying" the `Error`.